batch/test/test_batch.py

```diff
@@ -1296,3 +1296,15 @@ def test_submit_update_to_deleted_batch(client: BatchClient):
         assert err.status == 404
     else:
         assert False
+
+
+@skip_in_azure
```

Comment:
This should also work in azure, can we make a test fixture here that passes a us-east1 to the test method in gcp and eastus in azure? This PR is more testing that the region is provided in the status not necessarily that we can successfully schedule in different regions so it still makes sense as a test on azure.

Reply:
Let's also have the job command print the env variable and assert that it shows up in the job log

Reply:
I originally wanted to test it creates a VM in a different region, but I was concerned that this will continue to add to test times as it's another VM that we have to wait on to create and startup. Do you have preferences on what the test should do? I can increase the timeout on the batch tests to account for this. Also, it would have to be a highcpu instance because we've capped the number of standard instances so it will probably never be able to create a new instance before it times out.